source/MRViewer/MRRenderNameObject.cpp

```diff
@@ -32,7 +32,7 @@ void RenderNameObject::Task::earlyBackwardPass( const BackwardPassParams& backPa
             if ( ImGui::IsMouseDown( ImGuiMouseButton_Left ) )
                 isActive = true;
 
-            if ( ImGui::IsMouseClicked( ImGuiMouseButton_Left ) )
+            if ( getViewerInstance().getMenuPlugin()->uiManagerOwnsMouseHover() && ImGui::IsMouseClicked( ImGuiMouseButton_Left ) )
```

Comment:
should it be checked before set `isHover = true`?

Reply:
Nope, because `uiManagerOwnsMouseHover()` only starts returning true on mouse hover.

Reply:
then before `isActive = true;`?